pkg/metrics/metrics.go

```diff
@@ -131,6 +134,26 @@ var (
 	// LabelKind is the kind a label
 	LabelKind = "kind"
 
+	// LabelPath is the label for the API path
+	LabelPath = "api_path"
+
+	// LabelMethod is the label for the HTTP method
+	LabelMethod = "api_method"
+
+	// LabelAPIReturnCode is the HTTP code returned for that API path
+	LabelAPIReturnCode = "return_code"
+
+	// API interactions
+
+	// APIInteractions is the total time taken to process an API call made
+	// to the cilium-agent
+	APIInteractions = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Subsystem: Agent,
+		Name:      "api_process_time_seconds",
+		Help:      "API processing time stats labeled by API PATH",
```

Comment:
The description is not entirely clear, let's use `"Duration of processed API calls labeled by path, method and return code."`